bin/run.sh

```diff
@@ -18,7 +16,7 @@ case $1 in
     kinto start --ini $KINTO_INI
     ;;
   uwsgistart)
-    KINTO_INI=$KINTO_INI uwsgi --http :8888 --ini $KINTO_INI
+    uwsgi --http :$PORT --ini $KINTO_INI
```

Comment:
Should we set a default `PORT`? This is set in the `Dockerfile`, so most of the time we're covered, but in case someone is running this service without Docker, this script does make the assumption that there's a `PORT` env var set.

Similarly, why did we remove the default `KINTO_INI` env var? If we don't want to provide a default, can we display a message reminding the user to set that env var if one isn't set?

Reply:
I like the fact that all environment variables are listed and defaulted in the same place. Either Dockerfile or run.sh, but not both.

Also, using the `run.sh` outside the Docker context isn't something that we advocate anywhere, so I don't think we should worry about this.